langchain/assets/monitors/request_duration.json

```diff
@@ -2,11 +2,11 @@
   "version": 2,
   "created_at": "2023-07-21",
   "last_updated_at": "2023-07-21",
-  "title": "Request Duration Spike",
+  "title": "Request duration is increasing",
   "tags": [
     "integration:langchain"
   ],
-  "description": "Notify your team when requests made with LangChain have increased latency",
+  "description": "By default, LangChain will wait indefinitely for a response from the model provider. This monitor tracks the request duration to alert when requests made with LangChain have increased latency.",
```

Comment:
Hi @estherk15 could you please take a look at this description since I added it directly here and was not in the spreadsheet. 🙏